roadmap/implementers-guide/src/runtime/disputes.md

```diff
@@ -17,90 +19,351 @@ We account for these requirements by having the validity module handle two kinds
+The first is a remote roll-up of a **concluded dispute**. These are simply all attestations for the block, those against it, and the result of all (secondary) approval checks. A concluded remote dispute can be resolved in a single transaction as it is an open-and-shut case of a quorum of validators disagreeing with another.
 
-## Remote Disputes
+The second type of remote dispute is the **unconcluded dispute**. An unconcluded remote dispute is started by any validator using a `UnconcludedDisputeGossipNotificationMessage`:
 
-When a dispute has occurred on another fork, we need to transplant that dispute to every other fork. This poses some major challenges.
+When beginning a remote dispute, at least one escalation by a validator is required, but this validator may be malicious and desires to be slashed. There is no guarantee that the para is registered on this fork of the relay chain or that the para was considered available on any fork of the relay chain.
 
-There are two types of remote disputes. The first is a remote roll-up of a concluded dispute. These are simply all attestations for the block, those against it, and the result of all (secondary) approval checks. A concluded remote dispute can be resolved in a single transaction as it is an open-and-shut case of a quorum of validators disagreeing with another.
+The first step is the escalation, to notify peers of the unconcluded gossip.
 
-The second type of remote dispute is the unconcluded dispute. An unconcluded remote dispute is started by any validator, using these things:
+## Disputing an Unavailable Candidate
 
-- A candidate
-- The session that the candidate has appeared in.
-- Backing for that candidate
-- The validation code necessary for validation of the candidate.
-  > TODO: optimize by excluding in case where code appears in `Paras::CurrentCode` of this fork of relay-chain
-- Secondary checks already done on that candidate, containing one or more disputes by validators. None of the disputes are required to have appeared on other chains.
-  > TODO: validator-dispute could be instead replaced by a fisherman w/ bond
+In the case of malicious validator sending a dispute gossip message, with a block hash, that never became part of any parachain.
+
+If the candidate was not made available on another fork of the relay chain (as in it does not exist), the availability process will time out and the disputing validator will be slashed on this fork.
+As with other disputes, the escalation
+
+> TODO: is the escalation equivalent to the `DisputeGossipMessage` or is it something separate? Which information must be contained in that.
```

Comment:
I'd think so, we presumably want one gossip message type contains a candidate receipt, which it asserts is invalid, and also contains a backing vote for the candidate, which itself probably includes this candidate receipt.  

We'll always have a backing vote before any approval votes, so this should be general enough.  We need not validate the backing vote in its chain context, just validate that it contains a vote for that candidate.  In this way, our backing vote might also provide a bit of context that helps repeat approval vote types if they are a bit minimized somehow.